graphql-schemas/schema.yml

```diff
@@ -1430,6 +1431,16 @@ confs:
   - { name: region, type: string, isRequired: true }
   - { name: subnets, type: AWSSubnet_v1, isList: true }
 
+- name: AWSManagedVPC_v1
+  fields:
+  - { name: schema, type: string, isRequired: true }
+  - { name: path, type: string, isRequired: true }
+  - { name: labels, type: json }
+  - { name: name, type: string, isRequired: true }
+  - { name: description, type: string }
+  - { name: network, type: Network_v1, isRequired: true }
```

Comment:
Is the name network better than `cidr_block:` in `/aws/vpc-1.yml`? In my opinion Network is more vague and doesn't capture the core of the information. Another thing is that we will be having both vpc schemas for a while, would it better to have some parity between them?

Reply:
`network` is a reference to the network schema I prefer this because using `cidr_block` can be confusing for someone that didn't participated in the discussions about that.

A developer in the future can think it is weird to have `cidr_block` being a string in a schema and an object in another. More so for similar schemas.